### PR TITLE
Soft-Assert-Implementation

### PR DESCRIPTION
**Pull request by RichardSeverich**

### CODE REVIEW

File: src/test/java/org/fundacionjala/sfdc/stepdefinitions/AccountsSteps.java [hunk 1]
```diff
@@ -102,6 +104,6 @@ public void iClickOnDeleteFromAccount() {
     @Then("^the Account should not be displayed on Home Page$")
     public void theAccountShouldNotBeDisplayedOnHomePage() {
         AccountHome accountHome = new AccountHome();
-        assertFalse(accountHome.isDisplayedItem(map.get(AccountFormField.ACCOUNT_NAME)));
+        helper.getAssertion().assertFalse(accountHome.isDisplayedItem(map.get(AccountFormField.ACCOUNT_NAME)));
```
Comment (carledriss) [inline src/test/java/org/fundacionjala/sfdc/stepdefinitions/AccountsSteps.java:107]: please add a variable for helper.getAssertion()

Comment (RichardSeverich) [reply src/test/java/org/fundacionjala/sfdc/stepdefinitions/AccountsSteps.java]: It's okay, thanks for the observation.

Comment (RichardSeverich) [reply src/test/java/org/fundacionjala/sfdc/stepdefinitions/AccountsSteps.java:107]: Should I add a global or local variable?
Should I do this on all asserts?

Comment (RichardSeverich) [reply src/test/java/org/fundacionjala/sfdc/stepdefinitions/AccountsSteps.java:107]: OK I did it.

**Pull request merged**